includes/omise-wc-setting.php

```diff
@@ -0,0 +1,100 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+return array(
+	'enabled' => array(
+		'title'       => __( 'Enable/Disable', $this->gateway_name ),
+		'type'        => 'checkbox',
+		'label'       => __( 'Enable Omise Payment Module.', $this->gateway_name ),
+		'default'     => 'no'
+	),
+	'sandbox' => array(
+		'title'       => __( 'Sandbox', $this->gateway_name ),
```

Comment:
@nimid then, how about, just move it back this time that we can make a release for this PR, and implement this approach in that PR instead?